tensorflow_lite_support/python/task/processor/proto/classification_options_pb2.py

```diff
@@ -13,6 +13,74 @@
 # limitations under the License.
 """Classification options protobuf."""
 
+import dataclasses
+from typing import Any, List, Optional
+
+from tensorflow.tools.docs import doc_controls
 from tensorflow_lite_support.cc.task.processor.proto import classification_options_pb2
 
-ClassificationOptions = classification_options_pb2.ClassificationOptions
+_ClassificationOptions = classification_options_pb2.ClassificationOptions
+
+
+@dataclasses.dataclass
+class ClassificationOptions:
+  """Options for classification processor.
+
+  Attributes:
+    display_names_locale: The locale to use for display names specified through
+      the TFLite Model Metadata.
+    max_results: The maximum number of top-scored classification results to
+      return.
+    score_threshold: Overrides the ones provided in the model metadata. Results
+      below this value are rejected
+    class_name_allowlist: If non-empty, classifications whose class name is not
+      in this set will be filtered out. Duplicate or unknown class names are
+      ignored. Mutually exclusive with `class_name_denylist`
+    class_name_denylist: If non-empty, classifications whose class name is in
+      this set will be filtered out. Duplicate or unknown class names are
+      ignored. Mutually exclusive with `class_name_allowlist`.
+  """
+
+  score_threshold: Optional[float] = None
+  class_name_allowlist: Optional[List[str]] = None
+  class_name_denylist: Optional[List[str]] = None
+  display_names_locale: Optional[str] = None
+  max_results: Optional[int] = None
+
+  @doc_controls.do_not_generate_docs
+  def to_pb2(self) -> _ClassificationOptions:
+    """Generates a protobuf object to pass to the C++ layer."""
+    return _ClassificationOptions(
+      score_threshold=self.score_threshold,
+      class_name_allowlist=self.class_name_allowlist,
+      class_name_denylist=self.class_name_denylist,
+      display_names_locale=self.display_names_locale,
+      max_results=self.max_results)
+
+  @classmethod
+  @doc_controls.do_not_generate_docs
+  def create_from_pb2(
+      cls,
+      pb2_obj: _ClassificationOptions = _ClassificationOptions()
```

Comment:
@khanhlvg Do we drop the initialization for the parameter here as well?

Reply:
Yes, we should drop default option here and require users to specify one.

Reply:
@khanhlvg Updated it.